scene/gui/tree.cpp

```diff
@@ -28,6 +28,7 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 #include "tree.h"
+#include "../../core/os/main_loop.h"
```

Comment:
`core` is already in the include path, you can use `os/main_loop.h` directly. Also best put it next to the other Godot-related includes. Not sure why `<limits.h>` is there, but it should actually come last:
```
#include "tree.h"

#include "os/input.h"
#include "os/keyboard.h"
#include "os/main_loop.h"
#include "os/os.h"
#include "print_string.h"
#include "project_settings.h"
#include "scene/main/viewport.h"

#include <limits.h>
```